fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -188,12 +193,56 @@ private void incrementCallCount(FDBStoreTimer.Counts counter) {
         }
     }
 
+    public static boolean isSegmentInfo(String name) {
```

Comment:
Should these `isXFile` methods be `private`?

Reply:
I am afraid now. Looks these methods are accessed by code outside FDBDirectory.